lib/streaming-pipeline.js

```diff
@@ -106,22 +97,33 @@ module.exports = class StreamingPipeline extends Writable {
         }
 
         if (inputSignal.hasStart()) {
-            const outputContentTypes = inputSignal.getStart().getExpectedcontenttypesList();
+            const startSignal = inputSignal.getStart();
+            const outputContentTypes = startSignal.getExpectedcontenttypesList();
             if (this._startReceived) {
                 callback(new RiffError(
                     'error-streaming-too-many-starts',
-                    `start signal has already been received. Rejecting new start signal with content types [${outputContentTypes.join()}]`));
+                    'start signal has already been received. ' +
+                    `Rejecting start signal with: output content types: [${outputContentTypes.join(",")}], ` +
+                    `input names: [${this._streamMetadata.inputs.join(",")}], ` +
```

Comment:
this is the metadata for the original start and not the rejected start signal

Reply:
Oopsie, fixing now.